cmd/data/data.go

```diff
@@ -0,0 +1,27 @@
+package data
+
+import (
+	"github.com/spf13/cobra"
+	data_export "github.com/zitadel/zitadel-tools/cmd/data/export"
+	data_import "github.com/zitadel/zitadel-tools/cmd/data/import"
+)
+
+// Cmd represents the data root command
+var Cmd = &cobra.Command{
+	Use:   "data",
+	Short: "Import/Export data",
+}
+
+func init() {
+	issuer := Cmd.PersistentFlags().String("issuer", "", "issuer of your ZITADEL instance (in the form: https://<instance>.zitadel.cloud or https://<yourdomain>)")
+	api := Cmd.PersistentFlags().String("api", "", "gRPC endpoint of your ZITADEL instance (in the form: <instance>.zitadel.cloud:443 or <yourdomain>:443)")
+	insecure := Cmd.PersistentFlags().Bool("insecure", false, "disable TLS to connect to gRPC API (use for local development only)")
```

Comment:
I feel that these 3 command line options are quite redundant.

For example:

1. If the issuer is `http://localhost:8080`, you already know the API endpoint is `localhost:8080` and the connection is insecure.
2. If the issuer is `https://<instance>.zitadel.cloud`, you already know the API endpoint is `<instance>.zitadel.cloud:443` and the connection is secure.

For this example you'll need to parse the URL of the issuer and imply the the API configuration from there.

The reverse can also be applied:

1. A `--host` flag can point to the API and will become part of the issuer
2. The `--insecure` flag toggles `https` to `http` in front of the issuer. The API port toggles between `443` and `80` by default.
3. A `--port` flag sets / overrides the port suffix to the host for the issuer and API.

In the last example, if one would only set `<instance>.zitadel.cloud`, the default will create issuer `https://<instance>.zitadel.cloud` and API endpoint `<instance>.zitadel.cloud:443`.

For user friendlyness I would want to stick to one of the above solutions, you may pick one.